tools/c2chapel/test/no-args.2.good

```diff
@@ -0,0 +1,4 @@
+usage: c2chapel [-h] [--no-typedefs] [--debug] [--no-fake-headers]
```

Comment:
Does c2chapel still give the error in the original .good file? Why have both assuming it's deterministic?

Reply:
It depends on the exact versions of Python / Python libraries and these differ on different systems

Reply:
Ah, I see. Thanks.